### PR TITLE
feat: add ssh keys and certificates to admin-gui

### PR DESCRIPTION
**Pull request by HejdaJakub**
* My Profile - Authentication section was refactored. There is new submenu with two new sections (SSH Keys and Certificates).
* Component SSH Keys was already used in user-profile, so this component was extracted to to libs folder and reused in admin-gui.
* Page Certificates is created as a MVP, because we are not sure yet, if this page will be used by users (but now we want to move this functionality to the new gui from old gui). This page now displays table with only one attribute. Value of the certificate isn't displayed user-friendly, but as a MVP it should be sufficient.

### CODE REVIEW

File: libs/perun/components/src/lib/identities-page/identities-page.component.ts [hunk 1]
```diff
@@ -129,12 +137,16 @@ export class IdentitiesPageComponent implements OnInit {
           location.reload();
         } else if (result === 'OK') {
           this.notificator.showSuccess(
-            this.translate.instant('IDENTITIES.SUCCESSFULLY_ADDED') as string
+            this.translate.instant(
```
Comment (bodnara) [inline libs/perun/components/src/lib/identities-page/identities-page.component.ts:140]: Use the perun translate service instead.

File: libs/perun/components/src/lib/settings-ssh-keys/settings-ssh-keys.component.ts [hunk 2]
```diff
@@ -43,33 +43,37 @@ export class SettingsSSHKeysComponent implements OnInit {
...
     translateService
-      .get('SSH_KEYS.HEADER_COLUMN')
+      .get('SHARED_LIB.PERUN.COMPONENTS.SSH_KEYS.HEADER_COLUMN')
       .subscribe((value: string) => (this.headerColumnText = value));
   }
 
   ngOnInit(): void {
-    this.userId = this.store.getPerunPrincipal().userId;
+    if (window.location.pathname.startsWith('/myProfile')) {
+      this.userId = Number(window.location.pathname.split('/')[3]);
```
Comment (bodnara) [inline libs/perun/components/src/lib/settings-ssh-keys/settings-ssh-keys.component.ts:61]: Same question as before, can we use entity storage ?

**Review by bodnara — CHANGES_REQUESTED** [review]
When I use add button on the certificats page in admin-gui, it redirect to page not found.

File: libs/perun/components/src/lib/identities-page/identities-page.component.ts [hunk 3]
```diff
@@ -53,8 +54,15 @@ export class IdentitiesPageComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
-    this.userId = this.storage.getPerunPrincipal().userId;
-    this.displayCertificates = this.storage.get('display_identity_certificates') as boolean;
+    if (window.location.pathname.startsWith('/myProfile')) {
+      this.userId = Number(window.location.pathname.split('/')[3]);
```
[the remark this answers is not in the document]
Comment (HejdaJakub) [reply libs/perun/components/src/lib/identities-page/identities-page.component.ts:58]: The changes on the identities page were restored due to another final solution of the certificates page.

File: libs/perun/components/src/lib/identities-page/identities-page.component.ts (hunk 1 above)
Comment (HejdaJakub) [reply libs/perun/components/src/lib/identities-page/identities-page.component.ts:140]: The changes on the identities page were restored due to another final solution of the certificates page.

File: libs/perun/components/src/lib/settings-ssh-keys/settings-ssh-keys.component.ts (hunk 2 above)
Comment (HejdaJakub) [reply libs/perun/components/src/lib/settings-ssh-keys/settings-ssh-keys.component.ts:61]: Good point. Fixed.

**Review by bodnara — APPROVED** [review]

**Pull request merged**